std/algorithm/comparison.d

```diff
@@ -479,6 +479,15 @@ unittest
 Returns $(D val), if it is between $(D lower) and $(D upper).
 Otherwise returns the nearest of the two. Equivalent to $(D max(lower,
 min(upper,val))).
+
+Params:
+    val = The value to clamp.
+    lower = The lower bound of the clamp.
```

Comment:
It's to prevent ddoc from automatically highlighting it as a parameter (because the word happens to be identical to the symbol name of the parameter).